packages/core/src/types.ts

```diff
@@ -1799,7 +1799,9 @@ export interface Behavior<TEvent extends EventObject, TEmitted = any> {
 }
 
 export type EmittedFrom<T> = ReturnTypeOrValue<T> extends infer R
-  ? R extends ActorRef<infer _, infer TEmitted>
+  ? R extends Interpreter<infer _, infer __, infer ___, infer ____, infer _____>
```

Comment:
this might look a little bit strange because `Interpreter` implements `ActorRef` and `ActorRef` is already handled here - but, unfortunately, while it implements the `ActorRef`, its `send` method's signature is different and TS might not be able to actually treat it as `ActorRef` for the purpose of conditional types with `infer`